sdk/template/template/test/public/core-v1-test.spec.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I don't think this package is the right place for tests that use Azure storage.

Reply:
Hmmm, @timovv let's remove this and just keep the testing-recorder-new folder alive until the storage-queue is migrated.

Reply:
Yup.